NEWS.md

```diff
@@ -104,9 +88,6 @@ Language changes
     When `x` is global, `x::T = ...` and `global x::T` used to mean type assertion,
     but this syntax is now reserved for type declaration ([#964]).
 
-Command-line option changes
```

Comment:
we should actually mention that several of these are now deprecated and will be removed